todocli/cli.py

```diff
@@ -65,19 +69,32 @@
         Max: 99h
         
     morning
-        Tomorrow morning at 07:00 AM
+        Today at 07:00 AM if current time < 07:00 AM, otherwise tomorrow
         
     evening
-        Today at 06:00 PM if current time < 06:00 PM, otherwise tomorrow at 06:00 PM
+        Today at 06:00 PM if current time < 06:00 PM, otherwise tomorrow
+        
+    tomorrow
+        Tomorrow at 07:00 AM
         
     {hour}:{minute}
-        Today at {hour}:{minute}
+        Today at {hour}:{minute} if current time < {hour}:{minute}, otherwise tomorrow 
         e.g. 9:30, 09:30, 17:15
         
+    {hour}:{minute} am|pm 
+        Today at {hour}:{minute} am|pm  if current time < {hour}:{minute} am|pm, otherwise tomorrow
+        e.g. 9:30 am, 12:00 am, 10:15 pm
+        
     {day}.{month}. {hour}:{minute}
         The given day at {hour}:{minute}
         e.g. 24.12. 12:00
-        e.g. 7.4.   9:15"""
+        e.g. 7.4.   9:15
+        
+    {month}/{day}. {hour}:{minute} am|pm
+        The given day at {hour}:{minute} am|pm
+        e.g. 12/24 12:00 pm
+        e.g. 03/02   9:15
+        e.g. 3/2   09:15"""
```

Comment:
Is the only way to tell difference between two formats (day-month vs month-day) using `.` and `/`? If so, this could be very confusing. 

Just to be clear, is day-month the standard in Europe?

Reply:
Well, DD.MM.YYYY is the default in Germany... xD 
Unfortunately, date formats extremely vary around the globe. 
For a reference table see: https://en.wikipedia.org/wiki/Date_format_by_country

I see why this might confuse people... We probably need to decide what formats we want to allow.
Maybe we should even switch to a localized approach, I will do some research on if there are any libraries which are able to parse a date string based on the defined region of the PC.

Reply:
Sure. At this stage if its too complicated to detect region, we could just put the preferred format (month-day vs day-month) in a config file. Perhaps use `keys.yml` as a general configuration file and rename it.

Reply:
Okay, as this is going to be a bigger feature, I removed the input of MM/DD time strings for this PR. 
I'll work on that feature and commit another PR for that. 

So this PR should only include am / pm time strings now.